src/components/Pressable/PressableWithFeedback.js

```diff
@@ -46,10 +46,9 @@ const PressableWithFeedback = forwardRef((props, ref) => {
                         setDisabled(props.disabled);
                         return;
                     }
-                    onPress
```

Comment:
I don't think we need it. This prettier issue is now fixed here - https://github.com/Expensify/App/pull/19109/files

Reply:
merged main!